problem_builder/answer.py

```diff
@@ -261,12 +261,13 @@ def get_template(cls, template_id):
             return {'data': {'name': uuid.uuid4().hex[:7]}}
         return {'metadata': {}, 'data': {}}
 
-    def student_view_data(self):
+    def student_view_data(self, context=None):
         """
         Returns a JSON representation of the student_view of this XBlock,
         retrievable from the Course Block API.
         """
         return {
+            'weight': self.weight,
```

Comment:
@bradenmacdonald Done. I also added `id: self.name` for consistency with other blocks, but left `name: self.name` there as well, for backwards compatibility.